public/src/components/channelManagement/bannerTests/bannerTestsForm.tsx

```diff
@@ -57,8 +57,13 @@ const BannerTestsForm: React.FC<Props> = ({
   editMode,
 }: Props) => {
   const createTest = (name: string, nickname: string): void => {
-    const newTests = [...tests, createDefaultBannerTest(name, nickname)];
-    onTestsChange(newTests, name);
+    if (Object.keys(modifiedTests).length > 0) {
```

Comment:
I think we should refactor `modifiedTests` to be an array in a future PR.

Reply:
Yeah we can actually refactor to get rid of it entirely. It's a hangover from when mutliple tests could be worked on in one go. The new UX is that only one test can be worked on at any one time. Now we can just use a single `modifiedTest` object to keep track of the changes to a single test.

Reply:
👍 agreed, let's get rid of `modifiedTest` now that things are simpler